ql/src/java/org/apache/hadoop/hive/ql/ddl/table/misc/rename/AbstractAlterTableRenameAnalyzer.java

```diff
@@ -49,6 +55,11 @@ protected void analyzeCommand(TableName tableName, Map<String, String> partition
       setAcidDdlDesc(desc);
     }
     addInputsOutputsAlterTable(tableName, null, desc, desc.getType(), false);
+    String newDatabaseName = target.getDb() != null ? target.getDb() : table.getDbName(); // extract new database name from new table name, if not specified, then src dbname is used
+    Database newDatabase = getDatabase(newDatabaseName);
+    outputs.add(new WriteEntity(newDatabase, WriteEntity.WriteType.DDL_SHARED));
+    Table newTable = new Table(target.getDb(), target.getTable());
```

Comment:
I am not 100% sure, but shouldn't we propagate back from original table it's parameters, type, temp attribute, etc. See `DDLUtils.addDbAndTableToOutputs` & `SemanticAnalyzer.addDbAndTabToOutputs`

Reply:
That's a good point. I'll add them.

Reply:
why not just call
```` 
DDLUtils.addDbAndTableToOutputs(
        newDatabase, newTable
        table.getTableType(), table.isTemporary(), table.getParameters(), 
        outputs)`
````

Reply:
ack